matching-notification-rest/src/main/java/org/phenotips/matchingnotification/rest/MatchingNotificationResource.java

```diff
@@ -126,8 +129,8 @@ Response getMatches(@FormParam("minScore") @DefaultValue("0.5") double minScore,
     @POST
     @Consumes(MediaType.APPLICATION_JSON)
     @Produces(MediaType.APPLICATION_JSON)
-    @Path("/send-user-notifications")
-    Response sendUserNotification(@FormParam("matchId") String matchId,
+    @Path("/{matchId}")
+    Response sendUserNotification(@PathParam("matchId") Long matchId,
```

Comment:
In the issue description on JIRA I proposed to move all email-related functionality to `/matches/${matchId}/email` (both POST and GET). Lets wait until that is resolved before we merge the issue and finalize documentation

Reply:
@veronikaslc changes were approved